src/main/app/src/app/zaken/intake-afronden-dialog/intake-afronden-dialog.component.html

```diff
@@ -0,0 +1,24 @@
+<!--
+  ~ SPDX-FileCopyrightText: 2022 Atos
+  ~ SPDX-License-Identifier: EUPL-1.2+
+  -->
+
```

Comment:
Is het niet handiger om dit scherm op te bouwen met de <mfb-form> ?

Reply:
Besproken, omdat het om een dialog gaat, gaan we niet de < mfb-form > gebruiken